ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master__edge.yaml

```diff
@@ -23,13 +23,21 @@ base_images:
     name: test
     namespace: ocp-kni
     tag: dev-scripts
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.25.0
```

Comment:
hum, `1.25.0` doesn't seem to do there, we probably have to upgrade https://github.com/openshift/release/blob/master/core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci#L23-L27

Reply:
we use the `1.37.1` in assisted-service, we could align with that one here?

Reply:
:man_facepalming: 
Created https://github.com/openshift/release/pull/30887 and I'll ask dptp to approve it

Reply:
Instead I'll adjust it to ``1.37.1``
Waiting for https://github.com/openshift/assisted-installer/pull/510 to get in and I'll merge the changes